src/framework/components/light/component.js

```diff
@@ -1119,6 +1119,47 @@ class LightComponent extends Component {
         return this.light.shadowUpdateOverrides;
     }
 
+    /**
+     * Sets the number of shadow samples used for soft shadows when shadow type is
+     * {@link SHADOW_PCSS_32F}. Higher values result in smoother shadows but can decrease
+     * performance. Defaults to 16.
+     *
+     * @type {number}
+     */
+    set shadowSamples(value) {
+        this.light.shadowSamples = value;
+    }
+
+    /**
+     * Gets the number of shadow samples used for soft shadows.
+     *
+     * @type {number}
+     */
+    get shadowSamples() {
+        return this.light.shadowSamples;
+    }
+
+    /**
+     * Sets the number of blocker samples used for soft shadows when shadow type is
```

Comment:
I still don't really understand what a 'blocker' is from reading this.

Reply:
updated all these docs comments, thanks